include/rocksdb/db.h

```diff
@@ -353,10 +353,17 @@ class DB {
   virtual Status Put(const WriteOptions& options,
                      ColumnFamilyHandle* column_family, const Slice& key,
                      const Slice& value) = 0;
+  virtual Status Put(const WriteOptions& options,
+                     ColumnFamilyHandle* column_family, const Slice& key,
```

Comment:
SstFileWriter uses the order `Put(key, ts, value)`

(re-iterating to make sure this isn't lost in the discussion)

Reply:
Will address these comments once we agree on the overall direction. :)

Currently, this is not a blocker for us, since we are working on adding timestamp support to WriteCommittedTxn transactions. I hope we can get this resolved before
- release of RocksDB 7.0
- people start adding timestamp support for Merge, DeleteRange, etc.
- declaring (subset of) feature complete for customers
whichever comes first.